### ISSUE TITLE
LMDE4 Gimp.AppImage with plugins - G'MIC-Qt Preview error

### ISSUE THREAD

**Issue by Kuba-ZIL** [aferrero2707/gimp-appimage#58]
Hi friends,
I have a problem with gimp.appimage with plugins. The latest functional version for me is GIMP_AppImage-git-2.10.19-20200514-withplugins-x86_64.AppImage from this version all new versions (currently GIMP_AppImage-git-2.10.21-20200910-withplugins-x86_64.AppImage) when opening G'MIC-Qt plugins display an error message when generating a preview. As an example, there is an error in the "Apply external CLUT" section, but a similar error is generated in other sections.
```
Preview error: *** Error in ./fx_apply_haldclut_preview/gui_split_preview/*repeat/*local/*local/ *** 
Command 'apply_timeout': *** Error in ./fx_apply_haldclut_preview/gui_split_preview/*repeat/*local/*local/apply_timeout/*if/*local/ *** *** Error in ./fx_apply_haldclut_preview/gui_split_preview/*repeat/(...)/apply_timeout/*if/*local/*thread0/ *** Unknown command or filename '_'; did you mean '%'?
```

Any idea why this is happening and how to solve this problem?

Operating System: LMDE 4 (debbie) / Debian 4.19.132-1 /
Kernel: Linux 4.19.0-10-amd64
Architecture: x86-64

Thank you.

![gmic-qt 2020-09-13 07-34-38](https://user-images.githubusercontent.com/71205531/93011420-71b7ec80-f596-11ea-9c33-44a809f6d131.png)

**Comment by linuxgirl22** [comment]
Just to +1 this, as am having the same issues. Tried GIMP_AppImage-git-2.10.21-20200917-withplugins-x86_64.AppImage and then saw the update and moved to the latest. Most things work really well, but gmic mostly has these errors. 

EndeavourOS/Mate, LTS kernel, Ryzen7/RX5700XT, x86-64, running gtk system with no qt installed.

**Comment by linuxgirl22** [comment]
@Kuba-ZIL , I've tried GIMP_AppImage-git-2.10.13-20191010-withplugins-x86_64.AppImage and gmic is working perfectly in that! Thanks so much to the maintainers.

**Comment by Kuba-ZIL** [comment]
Version GIMP_AppImage-git-2.10.21-20201001-withplugins-x86_64.AppImage has the same problems with G'MIC-Qt plugins.

**Comment by waynesutton50** [comment]
The problem is still present in version GIMP_AppImage-git-2.10.23-20210129-withplugins-x86_64.AppImage

**Comment by Kuba-ZIL** [comment]
My solution for this problem is replace the file with an older version G´MIC-QT:  

_Decompiling .appimage file:_
**./GIMP_AppImage-git-2.10.23-20210129-withplugins-x86_64.AppImage --appimage-extract**

**Change the file to the latest functional version** - G´MIC-QT for GIMP 2.10 - Linux 64 bits **2.9.2_pre#200903**
[Download here >>](https://bit.ly/3pxE1QQ) 

_or old functional version - G´MIC-QT for GIMP 2.10 - Linux 64 bits 2.9.0_pre#200328_
[Download here >>](https://bit.ly/3cAadQz)

 _unzip and overwrite the original version_
**./squashfs-root/plug-ins/GMIC/plug-ins/gmic_gimp_qt/gmic_gimp_qt**

_Repacking .appimage file:_
**wget "https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage"
chmod +x appimagetool-x86_64.AppImage**

**./appimagetool-x86_64.AppImage squashfs-root**


Modified latest version "GIMP_AppImage-git-**2.10.23-20210129**-withplugins-x86_64.AppImage".
It is available for [download here >>](https://bit.ly/36VGgHl)

If you get an error message after running the latest version "GIMP_AppImage-git-2.10.23-20210129-withplugins-x86_64.AppImage": 

```
GEGL operation missing!
GIMP requires the GEGL operation "gegl:introspect".
This operation cannot be found. Check your
GEGL install and ensure it has been compiled
with any dependencies required for GIMP.
```


Install graphviz.

**sudo apt install graphviz**

I hope it helps.

**Comment by linuxgirl22** [comment]
Thanks @kuba-ZIL :-) Works perfectly!

**Comment by Kuba-ZIL** [comment]
**Updated version** - G´MIC-QT for GIMP 2.10 - Linux 64 bits **2.9.2_pre#200903**
[Download here >>](https://bit.ly/3pxE1QQ) 

**Modified latest version** - GIMP_AppImage-git-**2.10.23-20210218**-withplugins-x86_64.AppImage
[Download here >>](https://bit.ly/3uhtZa6)

**Comment by Flashwalker** [comment]
Yeah, a lot of gmic filters just doesn't works in this gimp appimage. 2.10.25 tested.
I found that gmic version 2.9.4 is shipped in Ubuntu repo and it works fine with GIMP 2.10.30 from Ubuntu repo.  
It would be nice to have that gmic here. But looks like that one is not statically compilled.

**Comment by Flashwalker** [comment]
@Kuba-ZIL, Am i right, your solution is just to downgrade the G´MIC-QT?
Or something else required?

**Comment by Flashwalker** [comment]
I found 2.9.4 here https://gmic.eu/files/linux/   
but qt version doesn't match
